jobs/forms.py

```diff
@@ -5,10 +5,17 @@
 
 class JobForm(forms.ModelForm):
 
-    company_name = forms.CharField(max_length=150, required=True)
+    company_name = forms.CharField(label='Company',
+                                   max_length=500,
+                                   required=True)
+    website = forms.URLField(label='Company website', required=True)
 
     class Meta:
         model = Job
-        fields = ('contact_email', 'title', 'description', 'city',
-                  'country', 'expiration_date')
-        exclude = ('company',)
+        fields = ('company_name', 'website', 'contact_email', 'title',
+                  'description', 'city', 'country')
```

Comment:
I wasn't sure about who should be permitted to change that. The person posting the job offer may want to change it as well - maybe because it is a permanent opening, or the opposite - it is valid only for a month or so. 

I can remove it for now and we can add it later if needed